master/painting.html

```diff
@@ -1222,14 +1224,45 @@ <h3 id="StrokeDashing">Dashing strokes: the <span class="property">'stroke-dasha
 <a>'stroke-dasharray'</a>: each dash and gap length is interpreted relative
 to the author's path length as specified by <a>'path/pathLength'</a>.</p>
 
+<p id="interpolationDashPattern"><a>'stroke-dasharray'</a> must use discrete
+interpolation if <var>start</var> or <var>end</var> value compute to <span class="prop-value">none</span>
+or are invalid.
```

Comment:
I haven't checked that this is what browsers actually do, but assuming it is. This looks good. I'd personally find the following a little easier to follow but it's up to you:

> 'stroke-dasharray' values are non-additive. For interpolation, 'stroke-dasharray' values are combined as follows:
> * If either |start| or |end| compute to 'none' or are invalid, they are combined using the discrete animation type.
> * Otherwise, repeat both dash patterns of |start| and |end| value list until the length of elements in both value lists match. Each item is then combined by computed value.

Reply:
Sounds better. Changed it to that.